### DISCUSSION TITLE
Used in UAV project

### DISCUSSION THREAD

**Discussion by royveshovda** [bluenviron/mediamtx#1340] [General]
Just wanted to give a shout out :loudspeaker:  and praise :pray:  for this repo, and show that I am using this service as a vital part of my UAV-project. :helicopter: 

I originally was build and using my own GStreamer-image. But with a footprint for the image of around 1GB. GStreamer (or to be fair; all the extra plugins) was not very suitable for my project.

Details can be found here: https://fancydrones.github.io/x500-cm4/
(Still missing a bit of documentation at the time of writing)

**Comment by aler9** [comment]
Hello, thank you very much for the appreciation, the server is used in some Drone projects along with [gomavlib](https://github.com/aler9/gomavlib). It works best when using the RTSP / UDP transport protocol, although there could be some missing frames due to network fluctuations (but maybe this is for the best in case of flying drones, since real time is more important than dropped frames).

[a comment by an automated account is not shown]
